src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+* Changed `OtlpLogExporter` to convert `ILogger` structured log inputs to `Attributes` in
+  OpenTelemetry (only active when `ParseStateValues` is `true` on `OpenTelemetryLoggerOptions`)
```

Comment:
```suggestion
* Changed `OtlpLogExporter` to convert `ILogger` structured log inputs to
  `Attributes` in OpenTelemetry (only active when `ParseStateValues` is `true`
  on `OpenTelemetryLoggerOptions`)
```

Looks like the CI failed on our markdownlint step. Line lengths are limited to 80 characters.